docs/releasenotes/release-1.127.md

```diff
@@ -1,35 +1,53 @@
-# v1.126.0
+* Special shout-outs to @acpana, @anhdle-sso, @cheftako, @ericpang777, @gemmahou, @haiyanmeng, @jasonvigil, @jingyih, @justinsb, @maqiuyujoyce, @nb-goog, @xiaoweim, @yuwenma, @ziyue-101 for their contributions to this release.
 
-** This version is not yet released; this document is gathering release notes for the future release **
+## Announcement 
 
-* ...
 
-* Special shout-outs to ... for their contributions to this release.
-TODO: list contributors with `git log v1.126.0... | grep Merge | grep from | awk '{print $6}' | cut -d '/' -f 1 | sort | uniq`
+## New Beta Resources (Direct Reconciler):
 
-## Announcement 
+* [`BigQueryAnalyticsHubListing`](https://cloud.google.com/config-connector/docs/reference/resource-docs/bigqueryanalyticshub/bigqueryanalyticshublisting)
 
-## New Beta Resources (Direct Reconciler):
+  * Manage the reference to a shared dataset that a publisher lists in a data exchange.
```

Comment:
Nit: I think it's easier to just list the fields we added. The information here feels redundant vs the field docs.  Ideally we would link them, but that seems less important than being precise about what we are adding.

Reply:
Oh, unless there are entirely new resources?  I would still just link to the resource (as you are doing) and not add the text.  "Manage the reference" made me think we were adding a ref field

Reply:
Could you be more specific about `list the fields we add`? This adds an entire new resource (beta), not a single field. Do you want me to list all the fields the resource contains? I added the link to the resource reference page which lists all the fields.

Reply:
Sorry, I was confused.  I don't think we should add the text description, because it duplicates the docs.  Better (but not required) to link to the docs.

Reply:
> Oh, unless there are entirely new resources? I would still just link to the resource (as you are doing) and not add the text. "Manage the reference" made me think we were adding a ref field

So adding the text to briefly explain what this resource is is from a discussion with the tech writer. We think it is a better form to swift the focus of the release note from a developer focus to a user focus. I can remove it here but may still want to keep it in the google release note

Reply:
Not sure I want to add all new fields for a new resource. That seems too verbose. I like the idea of linking the Google resource docs from the Google release notes. As far as I know we don't have OSS resource docs (Maybe we should) but till we do not sure that we have something we should link from these release notes.